exercism/beer-song/beer-song.js

```diff
@@ -0,0 +1,58 @@
+var BeerSong = function(){};
+
+  var answer = ''
+  var string1 = ' bottles of beer on the wall, '
+  var string2 = ' bottles of beer.\nTake one down and pass it around, '
+  var string3 = ' bottles of beer on the wall.\n'
+  var string11 = ' bottle of beer on the wall, '
+  var string31 = ' bottle of beer on the wall.\n'
+  var string22 = ' bottle of beer.\nTake it down and pass it around, '
+  var noMore = 'no more bottles of beer on the wall.\n'
+  var zeroBottles = 'No more bottles of beer on the wall, no more bottles of beer.\nGo to the store and buy some more, 99 bottles of beer on the wall.\n'
```

Comment:
look for patterns with repetition...if you are seeing yourself repeating code a lot then there is most likely a way for you to write a function to get through it faster